extensions/gdx-setup/res/com/badlogic/gdx/setup/resources/html/build.gradle

```diff
@@ -38,7 +38,8 @@ task startHttpServer () {
     }
 }
 
-task beforeRun(type: AppBeforeIntegrationTestTask, dependsOn: startHttpServer) {
+tasks.register('beforeRun', AppBeforeIntegrationTestTask) {

```

Comment:
This section in particular seems to entirely break HTML project imports when using gdx-setup. It's something about the integration testing using a task that causes it to fail, and I don't know what ways there are around it that still use `tasks.register()`. Using the `task` syntax from before this PR works fine.

Reply:
I know the task configuration is related because [gdx-liftoff](https://github.com/tommyettinger/gdx-liftoff) doesn't use it and doesn't have this problem. With the summer game jam coming up shortly, and HTML projects common there, it seems like any HTML jam submissions will have to be made with gdx-liftoff, an older gdx-setup, or an existing template.

Reply:
Let me revert this particular case?

It also seems unknown to me why this doesn't work...